boot/boot.js

```diff
@@ -1054,6 +1071,9 @@ $tw.Wiki = function(options) {
 				}
 				this.clearCache(title);
 				this.clearGlobalCache();
```

Comment:
Do we still need the GlobalCache()? ... I think it isn't used and it is cleared all the time you want to use it.

Reply:
@Jermolene  Or should we avoid clearing the global cache, if tiddlers are changed, that don't trigger the "dirty" flag?? Would this make sense?

Reply:
Hi @pmario I left the global cache mechanism (ie the stuff in boot.js) in place for backwards compatibility with plugins that may be using it. I also left the existing tag caching (ie the stuff in wiki.js) so that it's possible to turn the new indexing stuff off for testing/comparison purposes.

Reply:
Did you think about the "dirty" flag possibility. Would it be an option?

I did try to use the global cache. But it's unusable. It's always empty.... At least I couldn't make sense of it.

Reply:
The cache will only have anything in it if it is being used; try turning off the indexing and you'll see the cache being used instead.